mitmproxy/contentviews/graphql.py

```diff
@@ -30,6 +30,8 @@ def is_graphql_query(data):
 
 
 def is_graphql_batch_query(data):
+    if len(data) == 0:
+        return
     return isinstance(data, list) and isinstance(data[0], dict) and "query" in data[0]
```

Comment:
I think you should just add the condition here

```py
return isinstance(data, list) and len(data) > 0 and isinstance(data[0], dict) and "query" in data[0]
```

Reply:
If I do this then `isinstance(data[0], dict) and "query" in data[0]` will be checked and return `list index out of range`

Reply:
Ah, sorry, my bad. I'll change it your way.